torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4433,18 +4465,24 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                                 dtypes=[torch.cdouble]),
                    )),
     OpInfo('add',
+           ref=np.add,
+           variant_test_name='test_add',
```

Comment:
I'm confused about why we need a `variant_test_name` here? Won't this name the test `test_test_add_...`? Also applies to the other occurrences below.

Reply:
Sorry for the confusion, this is just for testing to only test add function. This will be removed during the final touches in the PR once design review is done.